base/complex.jl

```diff
@@ -1116,7 +1116,23 @@ big(::Type{Complex{T}}) where {T<:Real} = Complex{big(T)}
 big(z::Complex{T}) where {T<:Real} = Complex{big(T)}(z)
 
 ## Array operations on complex numbers ##
+"""
+    complex(A::AbstractArray)
+
+Return an array containing the complex analog of each entry in array `A`.
+
+Equivalent to `complex.(A)`, except that the return value may alias all or part
+of `A` in accordance with the behavior of `convert(T, A)` given output type `T`.
 
+# Examples
+```jldoctest
+julia> complex([1, 2, 3])
```

Comment:
This example is still in the original docstring and will show up twice when someone calls `?complex`.

Reply:
thanks. I'll update